frontend/src/hooks/api/actions/useUserFeedbackApi/useUserFeedbackApi.ts

```diff
@@ -2,15 +2,23 @@ import { IInternalBanner } from 'interfaces/banner';
 import useAPI from '../useApi/useApi';
 import { ProvideFeedbackSchema } from '../../../../openapi';
 
-const ENDPOINT = 'api/admin/user-feedback';
+const DIRECT_ENDPOINT = 'https://sandbox.getunleash.io/enterprise/feedback';
+const ENDPOINT = 'feedback';
```

Comment:
This will be used in the end for users. They will post feedback into their own instance.